### PR TITLE
WIP: *: Discuss impact and suggested next steps in condition messages

### PR DESCRIPTION
**Pull request by wking**
Make these more actionable, so users don't have to track down our Git docs.

### CODE REVIEW

File: pkg/cvo/availableupdates.go [hunk 1]
```diff
@@ -143,9 +147,10 @@ func (optr *Operator) getAvailableUpdates() *availableUpdates {
 
 func calculateAvailableUpdatesStatus(clusterID string, proxyURL *url.URL, tlsConfig *tls.Config, upstream, arch, channel, version string) ([]configv1.Update, configv1.ClusterOperatorStatusCondition) {
 	if len(upstream) == 0 {
+		baseConsoleURI := FIXME()
 		return nil, configv1.ClusterOperatorStatusCondition{
 			Type: configv1.RetrievedUpdates, Status: configv1.ConditionFalse, Reason: "NoUpstream",
-			Message: "No upstream server has been set to retrieve updates.",
+			Message: fmt.Sprintf("No upstream server has been set to retrieve updates.%s  Fix by setting spec.upstream in ClusterVersion to point to an update recommendation service, for example https://api.openshift.com/api/upgrades_info/v1/graph , %s/settings/cluster/", failedToRetrieveUpdatesImpact, baseConsoleURI),
```
Comment (vrutkovs) [inline pkg/cvo/availableupdates.go:153]: I don't think its worth pointing to a console, there are multiple ways how to update clusterversion - via `oc` or ArgoCD, so some might think the console is the only way to do that

Comment (wking) [reply pkg/cvo/availableupdates.go:153]: How do message-readers discover those ways?

Comment (vrutkovs) [reply pkg/cvo/availableupdates.go:153]: Probably a link to the docs would be best. Currently its split between [CLI](https://docs.openshift.com/container-platform/4.4/updating/updating-cluster-cli.html#update-upgrading-cli_updating-cluster-cli) and [web console](https://docs.openshift.com/container-platform/4.4/updating/updating-cluster.html#update-upgrading-web_updating-cluster)

Comment (wking) [reply pkg/cvo/availableupdates.go:153]: But also not available for some restricted-network clusters.  Still, docs links are easy, and we can pivot to console links later if there's interest.  Will reroll...

**Pull request closed without merging**